doc/quick-start.rst

```diff
@@ -200,3 +200,151 @@ read-write and read-only pool instances:
     >>> resp = conn.select('demo', 'AAAA', mode=tarantool.Mode.PREFER_RO)
     >>> resp
     - ['AAAA', 'Alpha']
+
+
+Receiving out-of-band messages
+----------------------------------
+
+Receiving out-of-band messages from a server that uses box.session.push 
+call is supported for methods: :meth:`~tarantool.Connection.call`, 
+:meth:`~tarantool.Connection.eval`, :meth:`~tarantool.Connection.select`, 
+:meth:`~tarantool.Connection.insert`, :meth:`~tarantool.Connection.replace`, 
+:meth:`~tarantool.Connection.update`, :meth:`~tarantool.Connection.upsert`, 
+:meth:`~tarantool.Connection.delete`.
+
+To work with out-of-band messages, 2 optional arguments are used in 
+the methods listed above:
+
+ * `on_push` - callback, launched with the received data for each out-of-band message. 
+Two arguments for this callback are expected:
+   
+   * the first is the received from an out-of-band message data.
+   * the second is `on_push_ctx`,  variable for recording the result of the callback work.
+ * `on_push_ctx` - result of the `on_push` work can be written to this variable.
```

Comment:
I would not be so categorical at the description. The callback can just use the value.

Reply:
Done